services/src/main/java/org/keycloak/services/resources/ThemeResource.java

```diff
@@ -75,13 +75,14 @@ public class ThemeResource {
     @GET
     @Path("/{version}/{themeType}/{themeName}/{path:.*}")
     public Response getResource(@PathParam("version") String version, @PathParam("themeType") String themType, @PathParam("themeName") String themeName, @PathParam("path") String path) {
-        if (!version.equals(Version.RESOURCES_VERSION)) {
+        final Theme.Type type = getThemeType(themType);
```

Comment:
Perhaps we can fix this typo while we're at it?
```suggestion
        final Theme.Type type = getThemeType(themeType);
```

Reply:
Yep! Done!